charts/external-nfs/template/external-nfs-pv.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
This is not a valid Helm chart. Relocate this to /nfs-samples.

Reply:
moved to nfs-samples